pkg/services/teams.go

```diff
@@ -139,7 +140,12 @@ func (n *TeamsNotification) GetTemplater(name string, f texttemplate.FuncMap) (T
 }
 
 type TeamsOptions struct {
-	RecipientUrls map[string]string `json:"recipientUrls"`
+	RecipientUrls       map[string]string `json:"recipientUrls"`
+	InsecureSkipVerify  bool              `json:"insecureSkipVerify"`
+	MaxIdleConns        int               `json:"maxIdleConns"`
+	MaxIdleConnsPerHost int               `json:"maxIdleConnsPerHost"`
+	MaxConnsPerHost     int               `json:"maxConnsPerHost"`
+	IdleConnTimeout     time.Duration     `json:"idleConnTimeout"`
```

Comment:
We can't use time.Duration for parsing time values - we will need to use a string and then parse the actual `time.Duration` by doing something like this 

```golang
	idleConnTimeout, err := time.ParseDuration(s.opts.IdleConnTimeout)
	if err != nil {
		return errors.Wrap(err, "failed to parse idle connection timeout")
	}
```

and then passing in `idleConnTimeout` to the `NewTransport` function.